sunkit_pyvista/plotter.py

```diff
@@ -218,9 +219,13 @@ def plot_map(self, m, clip_interval: u.percent = None, **kwargs):
         else:
             clim = [0, 1]
         cmap = self._get_cmap(kwargs, m)
-        self.plotter.add_mesh(map_mesh, cmap=cmap, clim=clim, **kwargs)
+        color = kwargs.pop('color', None)
```

Comment:
Sorry, I was wrong in my earlier comment - we shouldn't allow color as a keyword argument to `plot_map`, as it makes no sense to plot a map in a single color. Instead just leave this line in, add a warning below in the `if` block, and don't add color to the field array.

Reply:
Alright, I'll make this change.